core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java

```diff
@@ -169,7 +169,7 @@ private static Properties readProperties(URL url) {
         try (InputStream inputStream = url.openStream()) {
             properties.load(inputStream);
         } catch (FileNotFoundException e) {
-            log.trace("Testcontainers config override was found on {} but the file was not found", url, e);
+            log.warn("Testcontainers config override was found on {} but the file was not found", url, e);
```

Comment:
this does not seem to fix the issue described in #2253.
If I understood it correctly, @vmassol was talking about **not** displaying the stacktrace but the only the error message, to make the logs less verbose

Reply:
ugh. My bad. removing "e" should help then ?

Reply:
please follow the issue's description, it is very well described and suggests a solution

Reply:
Understood. Will do.

Reply:
Resubmitted